orttraining/orttraining/python/training/utils/__init__.py

```diff
@@ -2,12 +2,22 @@
 # Licensed under the MIT License.
 # __init__.py
 
-from .torch_io_helper import (
-    PrimitiveType,
+from onnxruntime.training.utils.torch_io_helper import (
+    ORTModelInputOutputSchemaType,
     ORTModelInputOutputType,
+    PrimitiveType,
     _TensorStub,
-    ORTModelInputOutputSchemaType,
-    get_schema_for_flatten_data,
     flatten_data_with_schema,
+    get_schema_for_flatten_data,
     unflatten_from_data_and_schema,
-)  # noqa: F401
+)
+
+__all__ = [
+    "PrimitiveType",
+    "ORTModelInputOutputType",
+    "_TensorStub",
```

Comment:
Was it intended to expose this private class? If used outside of this module, should it be marked public?

Reply:
Hmmm, it is only used for ut, let me remove it from module's all, and reference with other way in the unit test.